cni-plugin/pkg/install/install.go

```diff
@@ -401,6 +401,11 @@ func writeCNIConfig(c config) {
 		logrus.Fatal(err)
 	}
 
+	err = os.Chmod(path, perm) // safeguarding since WriteFile does not change existing file's permissions
```

Comment:
```suggestion
        // Safeguarding since WriteFile does not change existing file's permissions.
	err = os.Chmod(path, perm) 
```

Reply:
Just a style point - we try to keep comments above rather than on the same line.

Reply:
Done